code/modules/unit_tests/_unit_tests.dm

```diff
@@ -23,7 +23,6 @@
 #include "anchored_mobs.dm"
 #include "bespoke_id.dm"
 #include "binary_insert.dm"
-//#include "chain_pull_through_space.dm" - Removed, as crosslinked levels are not used.
```

Comment:
We should probably think about making a unit test, can be a followup PR, I can even probably try my hand at making it.